cypress/support/cypress-support/src/globalSetter.js

```diff
@@ -1,19 +1,20 @@
 const UTILS = require('./utils');
 const CONSTANTS = require('../../constants/constants');
+const fcsSetterStack = require('./fcsSetterStack');
 import { apiObject } from '../../appObjectConfigs.js';
 
 global.setterSuccess = async (message = 'Setter Method is success') => {
-  const methodName = UTILS.getEnvVariable(CONSTANTS.REQUEST_OVERRIDE_METHOD);
   const params = UTILS.getEnvVariable(CONSTANTS.REQUEST_OVERRIDE_PARAMS);
   const response = { jsonrpc: '2.0', result: null, id: 0 };
   // Validating the response
-  await validateResponse(methodName, params, response);
-  console.log(`[${methodName}] ${message}`);
+  await validateResponse(params, response);
+  cy.log(`${message}`);
   return response;
 };
 
 global.setterFailure = (message, error) => {
-  const methodName = UTILS.getEnvVariable(CONSTANTS.REQUEST_OVERRIDE_METHOD);
+  fcsSetterStack.popMethod();
```

Comment:
Thinking more about this, wouldn't we not want to pop here but in cy.sendMessageToPlatforms()?

The actual logic processing the request shouldn't be responsible for the stack. SMTP should be responsible and the logic inside (Request overrides, FCS setters, etc) should just be referencing the stack.